session/session.go

```diff
@@ -941,6 +941,12 @@ func (s *session) doCommitWithRetry(ctx context.Context) error {
 	}
 	err = s.doCommit(ctx)
 	if err != nil {
+		// polish the Write Conflict error message
+		newErr := s.tryReplaceWriteConflictError(err)
```

Comment:
Why not replace the error in function `newWriteConflictError`? If so, the writeconfilict error everywhere can be unified

Reply:
Because information schema is needed to find the table name which is not available in `newWriteConflictError`